tests/database/migrations/2014_10_12_000000_create_exponent_push_notification_interests_table.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
why we can't use https://github.com/RonasIT/laravel-exponent-push-notifications/blob/68b7f0b8f66c63b7abcacc93dc7652c07225a3ef/migrations/create_exponent_push_notification_interests_table.php.stub in tests?